docs/Tutorials/Examples/Private-Network-Example.md

```diff
@@ -238,169 +252,172 @@ created during the genesis of this private test network.
 
-Save the configuration and return to the MetaMask main screen. Your current network is now set to
-the private network RPC node.
+In the [Block Explorer](http://localhost:25000), search for the new test account by clicking on the :mag: and pasting
+the test account address into the search box.
 
-[Import one of the existing accounts above into MetaMask] using the corresponding private key.
+The new test account displays with a zero balance.
 
-!!!note
-
-    In this tutorial, we don't need to secure the keys, because we're using a private test network
-    to send valueless Ether. In a production environment, be sure to secure your accounts on the
-    main Ethereum network (MainNet).
-
-When you're finished importing an existing account, [create another account from scratch] to send
-Ether to.
+[Send test Ether](https://metamask.zendesk.com/hc/en-us/articles/360015488991-Sending-Ether-New-UI-)
+from the first test account (containing test Ether) to the new test account (which has a zero balance).
 
-In MetaMask, select the new account and copy the account address by clicking the **&hellip;**
-gadget and selecting **Copy Address to clipboard**.
+!!!tip
 
-In the block explorer, search for the new account by clicking on the magnifying glass and pasting
-the account address into the search box. The account displays with a zero balance.
+    You can use a zero gas price here as this private test network is a [free gas network](../../HowTo/Configure/FreeGas.md)
+    but the maximum amount of gas that can be used (the gas limit) for a value transaction must be at least 21000.
 
-[Send Ether](https://metamask.zendesk.com/hc/en-us/articles/360015488991-Sending-Ether-New-UI-)
-from the first account (containing Ether) to the new account (which has a zero balance).
+Refresh the Block Explorer page in your browser displaying the target test account.
 
-Refresh the browser page displaying the new account. The updated balance displays and reflects the
-transaction completed using MetaMask.
+The updated balance displays and reflects the transaction completed using MetaMask.
 
-## Smart contract and DApp usage with the blockchain network
+## Smart contract and dapp usage
 
-We are going to use the
-[PetShop tutorial on Truffle website](https://truffleframework.com/tutorials/pet-shop) with a few
-modifications, so we use the private network in this tutorial as the blockchain network.
+You are now going to use a demo dapp called Pet Shop and provided by [Truffle](https://www.trufflesuite.com/tutorials/pet-shop).
 
-### Prerequisites
+This dapp will run a local website using Docker and use smart contracts on the blockchain that you started in the previous steps.
 
-<!-- vale off -->
-* [Node.js v6+ LTS and npm](https://nodejs.org/en/) (comes with Node)
-<!-- vale on -->
-* Import an account into MetaMask using the private key
-  `0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3`. The following steps use
-  this account.
```

Comment:
This step to import the specified account needs to be re-added to the tutorial.

Reply:
Why?

Reply:
Oh - I suppose you need an account with test ether to pay the gas fees - but I see that if you imported one of the genesis accounts during the transaction step then you don't need another account. In that case, you need to edit line 408 where this account is referenced.

Reply:
I made a change on line 408 to specify the use of a test account.
But also because the network is a free gas network, you could run the tx with any account, you don't even need ether on it do adopt a pet as it's a smart contract call without ether transfer. So giving the correct amount of max gas to be used is required but it can have a zero gas price thus no need of a positive balance on the account. But not to confuse users, asking to use one of the test accounts is fine.